tailwind.config.js

```diff
@@ -15,6 +15,8 @@ const config = {
                 yellow: '#FFC32A',
                 green: '#46B9AD',
                 orange: '#E36131',
+                'input-gray': '#F3F3F3',
```

Comment:
Tailwind has a color code `#F3F4F5` for `gray-100`, which is acceptable to use. However, you can continue with your own method if you want a more precise color representation. The decision is yours.